src/components/GrowlNotification/index.js

```diff
@@ -55,65 +43,80 @@ class GrowlNotification extends Component {
      * @param {String} type
      * @param {Number} duration
      */
-    show(bodyText, type, duration) {
-        this.setState(
-            {
-                bodyText,
-                type,
-            },
-            () => {
-                this.fling(0);
-                setTimeout(() => {
-                    this.fling(INACTIVE_POSITION_Y);
-                }, duration);
-            },
-        );
-    }
+    const show = useCallback((text, growlType, growlDuration) => {
+        setBodyText(text);
+        setType(growlType);
+        setDuration(growlDuration);
+    }, []);
 
     /**
      * Animate growl notification
      *
      * @param {Number} val
      */
-    fling(val = INACTIVE_POSITION_Y) {
-        Animated.spring(this.state.translateY, {
+    const fling = useCallback((val = INACTIVE_POSITION_Y) => {
```

Comment:
No child component is wrapped with memo, so useCallback won't do any optimization between renders

Reply:
It is being used in the `useEffect` so it must be a stable function, so it needs the `useCallback`

Reply:
Good point! I think this should be mentioned on React core docs